minecraft/protocol/packet/simple_event.go

```diff
@@ -14,7 +14,7 @@ const (
 // additional event data. The event is typically used by the client for telemetry.
 type SimpleEvent struct {
 	// EventType is the type of the event to be called. It is one of the constants that may be found above.
-	EventType int16
+	EventType uint16
```

Comment:
What is the source of this one? The docs doesn't say a type and everyone else encodes this as signed